Makefile

```diff
@@ -2,7 +2,7 @@ VERSION := $(shell git describe --tags)
 DIST_DIRS := find * -type d -exec
 
 build:
-	go build -o glide -ldflags "-X main.version=${VERSION}" glide.go
+	go build -o glide -ldflags "-X main.version=$(VERSION)" glide.go
```

Comment:
> $(VERSION) tells the shell script to execute VERSION and return the result of that.

No `$(VERSION)` is expanded my `make` before the shell even sees it. `${VERSION}` is also expanded by `make` (before being seen by the shell), but as it is the same syntax as the shell, this is confusing (readers may expect like you did that the expansion is done by the shell) and must be avoided. To really let the shell see the reference to a `VERSION` environment variable, you have to write `$${VERSION}`, but this is not portable with non-Unix shells (for example, on Windows) while the expansion by `make` is.

You should read a [`make` manual](https://gnu.org/software/make/manual/html_node/Reference.html#Reference).